napari/layers/tracks/tracks.py

```diff
@@ -574,17 +574,40 @@ def _norm(p):
         # actually set the vertex colors
         self._track_colors = colormap.map(vertex_properties)
 
-    @property
-    def track_connex(self) -> np.ndarray:
-        """vertex connections for drawing track lines"""
-        return self._manager.track_connex
-
     @property
     def track_colors(self) -> np.ndarray:
         """return the vertex colors according to the currently selected
         property"""
         return self._track_colors
 
+    @track_colors.setter
+    def track_colors(self, colors: np.ndarray) -> None:
+        """update vertex colors given a array of colors or a single colors,
+        colors must be in rgb and between 0 and 1
+        """
```

Comment:
It would be nice if this docstring explained how the colors should be formatted.

Reply:
Updated!